.changeset/slimy-monkeys-love.md

```diff
@@ -0,0 +1,7 @@
+---
+'@microsoft/atlas-css': patch
+'@microsoft/atlas-js': patch
+'@microsoft/atlas-site': patch
+---
+
+Updating checkbox component with validation colors and sizes. Updating form behavior validation so it validates checkboxes correctly and provides an option to hide an error banner. Updating webiste documentation to reflect recent changes.
```

Comment:
These should be different changesets I believe: 
Updating checkbox component with validation colors and sizes. - minor change
Updating form behavior validation so it validates checkboxes correctly and provides an option to hide an error banner. - patch
Updating **website** documentation to reflect recent changes. - minor change

Reply:
1,3 could be the same

Reply:
Updated